### PR TITLE
i18n(es): Translate `cms.md` 

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Translated `cmd.md` to spanish  based on https://github.com/withastro/docs/pull/1975

### CODE REVIEW

**Review by delucis — COMMENTED** [review]
Woah! That was quick 🤯 

Might be good to include the additions to `nav.ts` and `ui.ts` too:

https://github.com/withastro/docs/blob/8d49a475ca57574baa5bb9e75231a266db409f3d/src/i18n/en/ui.ts#L48-L49

☝️ Translate this heading in `src/i18n/es/ui.ts`

https://github.com/withastro/docs/blob/8d49a475ca57574baa5bb9e75231a266db409f3d/src/i18n/en/nav.ts#L60

☝️ Translate this navigation sidebar label in `src/i18n/es/nav.ts`, something like:

```js
'guides/cms': 'TRANSLATED LABEL',
```

File: src/pages/es/guides/cms.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Usa un CMS con Astro
+description: Cómo usar un CMS para agregar contenido a Astro
+layout: ~/layouts/MainLayout.astro
+setup: |
+  import CMSGuidesNav from '~/components/CMSGuidesNav.astro';
+i18nReady: true
+---
+
+**Ready to connect a Headless CMS to your Astro project?** Follow one of our guides to integrate a CMS.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/cms.md:10]:
```suggestion
```

File: src/pages/es/guides/cms.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+**¿Listo para conectar un Headless CMS a su proyecto Astro?** Siga una de nuestras guías para integrar un CMS.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/cms.md:11]:
```suggestion
**¿Listo para conectar un Headless CMS a tu proyecto Astro?** Sigue una de nuestras guías para integrar un CMS.
```

File: src/pages/es/guides/cms.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Guías de CMS
+
+<CMSGuidesNav />
+
+Tome en cuenta que muchas de estas páginas son **talones**: ¡Son colecciones de recursos esperando su contribución!
+
+## ¿Por qué usar un CMS?
+
+Un sistema de gestión de contenido le permite escribir contenido y administrar activos fuera de su proyecto Astro.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/cms.md:21]:
```suggestion
Un sistema de gestión de contenido te permite escribir contenido y administrar activos fuera de tu proyecto Astro.
```

File: src/pages/es/guides/cms.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+Esto desbloquea nuevas funciones para trabajar con contenido. La mayoría de los CMS te brindan un editor de contenido visual, la capacidad de especificar tipos de contenido estándar y una forma de colaborar con otros.
+
+Un CMS puede ser útil para el contenido que sigue una estructura particular, a menudo te brinda una experiencia de panel de control y herramientas de edición WYSIWYG. Puedes usar un CMS para escribir publicaciones de blog usando el editor de texto enriquecido de un CMS en lugar de archivos Markdown. O puedes usar un CMS para mantener listados de productos para una tienda en línea, haciendo que ciertos campos sean obligatorios para evitar listados incompletos.
+
+Tu proyecto Astro luego puede recuperar su contenido de su CMS y mostrarlo, donde y como quieras en tu sitio.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/cms.md:27]:
```suggestion
Tu proyecto Astro luego puede obtener el contenido de tu CMS y mostrarlo, donde y como quieras en tu sitio.
```

File: src/pages/es/guides/cms.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+
+## ¿Qué CMSes funcionan bien con Astro?
+
+Debido a que Astro se ocupa de la _presentación_ de tu contenido, tu vas querer elegir un CMS _headless_, como los de la lista anterior. Esto significa que el CMS le ayuda a escribir su contenido, pero no genera un sitio que lo muestre. En su lugar, recupera los datos de contenido y los usa en tu proyecto Astro.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/cms.md:32]:
```suggestion
Debido a que Astro se ocupa de la _presentación_ de tu contenido, procura elegir un CMS _headless_, como los de la lista anterior. Esto significa que el CMS te ayuda a escribir tu contenido, pero no genera un sitio que lo muestre. En su lugar, obtiene los datos de tu contenido y los usa en tu proyecto Astro.
```

File: src/pages/es/guides/cms.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+Algunos CMSes headless, como Storyblok, proporcionan una [integración](/es/guides/integrations-guide/) de Astro que ayuda a recuperar el contenido específicamente para un sitio Astro. Otros proporcionan un SDK de JavaScript, una biblioteca que instala y usa para recuperar tu contenido remoto.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/cms.md:34]:
```suggestion
Algunos CMSes headless, como Storyblok, proporcionan una [integración](/es/guides/integrations-guide/) de Astro que te ayuda a recuperar el contenido específicamente para un sitio Astro. Otros proporcionan un SDK de JavaScript, una biblioteca que instalas y usas para recuperar tu contenido remoto.
```

File: src/pages/es/guides/cms.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+## ¿Puedo usar Astro sin un CMS?
+
+¡Si! Astro proporciona formas integradas de [autorizar contenido](/es/guides/content/), incluido el soporte para páginas de Markdown.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/cms.md:38]:
```suggestion
¡Si! Astro proporciona formas integradas para [crear contenido](/es/guides/content/), incluyendo el soporte para páginas de Markdown.
```

**Review by kevinzunigacuellar — COMMENTED** [review]
increiblemente rapido 🇪🇸, deje algunos comentarios. Una duda no entendi el termino WYSIWYG quizas debamos explicarlo en parentesis?

**Comment by dreyfus92** [comment]
> increiblemente rapido 🇪🇸, deje algunos comentarios. Una duda no entendi el termino WYSIWYG quizas debamos explicarlo en parentesis?

WYSIWYG = What You See Is What You Get

File: src/pages/es/guides/cms.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+i18nReady: true
+---
+
+**¿Listo para conectar un Headless CMS a tu proyecto Astro?** Sigue una de nuestras guías para integrar un CMS.
+
+## Guías de CMS
+
+<CMSGuidesNav />
+
+Tome en cuenta que muchas de estas páginas son **talones**: ¡Son colecciones de recursos esperando su contribución!
```
Comment (agustinmulet) [inline src/pages/es/guides/cms.md:16]:
```suggestion
Ten en cuenta que muchas de estas páginas son **talones**: ¡Son colecciones de recursos esperando tu contribución!
```

File: src/pages/es/guides/cms.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+## ¿Por qué usar un CMS?
+
+Un sistema de gestión de contenido te permite escribir contenido y administrar activos fuera de tu proyecto Astro.
+
+Esto desbloquea nuevas funciones para trabajar con contenido. La mayoría de los CMS te brindan un editor de contenido visual, la capacidad de especificar tipos de contenido estándar y una forma de colaborar con otros.
+
+Un CMS puede ser útil para el contenido que sigue una estructura particular, a menudo te brinda una experiencia de panel de control y herramientas de edición WYSIWYG(Lo que ves es lo que obtienes). Puedes usar un CMS para escribir publicaciones de blog usando el editor de texto enriquecido de un CMS en lugar de archivos Markdown. O puedes usar un CMS para mantener listados de productos para una tienda en línea, haciendo que ciertos campos sean obligatorios para evitar listados incompletos.
```
Comment (agustinmulet) [inline src/pages/es/guides/cms.md:24]:
```suggestion
Un CMS puede ser útil para el contenido que sigue una estructura particular, a menudo te brinda una experiencia de panel de control y herramientas de edición WYSIWYG (Lo que ves es lo que obtienes). Puedes usar un CMS para escribir publicaciones de blog usando el editor de texto enriquecido de un CMS en lugar de archivos Markdown. O puedes usar un CMS para mantener listados de productos para una tienda en línea, haciendo que ciertos campos sean obligatorios para evitar listados incompletos.
```

File: src/pages/es/guides/cms.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+## ¿Qué CMSes funcionan bien con Astro?
+
+Debido a que Astro se ocupa de la _presentación_ de tu contenido, procura elegir un CMS _headless_, como los de la lista anterior. Esto significa que el CMS te ayuda a escribir tu contenido, pero no genera un sitio que lo muestre. En su lugar, obtiene los datos de tu contenido y los usa en tu proyecto Astro.
+
+Algunos CMSes headless, como Storyblok, proporcionan una [integración](/es/guides/integrations-guide/) de Astro que te ayuda a recuperar el contenido específicamente para un sitio Astro. Otros proporcionan un SDK de JavaScript, una biblioteca que instalas y usas para recuperar tu contenido remoto.
+
+## ¿Puedo usar Astro sin un CMS?
+
+¡Si! Astro proporciona formas integradas para [crear contenido](/es/guides/content/), incluyendo el soporte para páginas de Markdown.
```
Comment (agustinmulet) [inline src/pages/es/guides/cms.md:37]:
```suggestion
¡Sí! Astro proporciona formas integradas para [crear contenido](/es/guides/content/), incluyendo el soporte para páginas de Markdown.
```

**Review by agustinmulet — CHANGES_REQUESTED** [review]
Pequeños cambios sugeridos. Buen trabajo! 😤

**Comment by kevinzunigacuellar** [comment]
gracias @agustinmulet y @dreyfus92 😇

**Pull request merged**